app/js/preloads/follow.js

```diff
@@ -69,27 +67,26 @@ ipcRenderer.on("focus-first-text-input", async () => {
     }
 })
 
-const sendFollowLinks = () => {
-    if (!inFollowMode) {
-        return
-    }
-    allElements = [...document.querySelectorAll("*")]
-    const allLinks = []
+const getLinkFollows = allLinks => {
     // A tags with href as the link, can be opened in new tab or current tab
     allLinks.push(...allElementsBySelectors("url", urls))
-    allElements.filter(el => el.matches("a *")).forEach(el => {
-        const clickableSubElement = parseElement(el, "url")
-        const baseLink = allElements.find(
-            l => l.nodeName === "A" && l.contains(el))
-        if (baseLink && !parseElement(baseLink) && clickableSubElement) {
-            clickableSubElement.url = baseLink.href || ""
-            allLinks.push(clickableSubElement)
+    document.querySelectorAll("a *").forEach(el => {
+        const baseLink = el.closest("a")
+        if (baseLink && !allLinks.includes(baseLink)) {
```

Comment:
This check isn't correct and never matches, `allLinks` is a list of parsed links, while `baseLink` is a DOM element. Please double check with the diff from the code on master. You can test this on the newtab page, there should only be 1 follow element per link.

Reply:
Ah, what was I thinking! I've also added the explanation from your other comment.